app/controllers/user_controller.py

```diff
@@ -1,18 +1,40 @@
+from http import HTTPStatus
+
+from app.configs.database import db
+from app.models.user_model import UserModel
+from flask import jsonify, request, session
+from flask_jwt_extended import (create_access_token, get_jwt_identity,
+                                jwt_required)
+from sqlalchemy.orm import Session
+
+
```

Comment:
Use `isort .` and then `black .` in the terminal, we might run into some conflicts regarding the imports at some point if we don't do that.

Reply:
Agreed!